indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -297,6 +297,7 @@
       font-style: italic;
       font-size: 0.9em;
       color: $dark-gray;
+      word-wrap: break-word;
```

Comment:
Apparently, this property has been renamed to `overflow-wrap`: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
sure I can change to this, I just chose the one that appeared more frequently in the code base

Reply:
I'm also fine either way, just wanted to mention that :)